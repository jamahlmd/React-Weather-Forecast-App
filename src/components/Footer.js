import React from 'react';
import { Row, Col } from 'antd';
import propTypes from 'prop-types';

const Footer = ({text}) => (

    <div>
        <Row style={styles.footerStyles}>
            <Col md={{span: 6, offset:18}} xs={{span: 10, offset:14}}>
                <h5>{text}</h5>
            </Col>
        </Row>
    </div>

);

const styles = {
    footerStyles: {
        backgroundColor: '#ADD6FF',
        paddingTop: '6px',
        paddingBottom: '3px'
    }
};

Footer.propTypes = {
  text: propTypes.string.isRequired
};

export default Footer;