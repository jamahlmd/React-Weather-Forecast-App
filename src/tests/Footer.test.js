import React from 'react';
import Footer from '../components/Footer';

const wrapper = shallow(<Footer text={"Test text"}/>);


test('Should render Footer correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
});



test('Should render text correctly', () => {
    expect(wrapper.find('h5').text()).toBe('Test text');
});