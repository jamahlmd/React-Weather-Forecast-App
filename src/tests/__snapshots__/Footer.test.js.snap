// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer correctly 1`] = `
<div>
  <Row
    gutter={0}
    style={
      Object {
        "backgroundColor": "#ADD6FF",
        "paddingBottom": "3px",
        "paddingTop": "6px",
      }
    }
  >
    <Col
      md={
        Object {
          "offset": 18,
          "span": 6,
        }
      }
      xs={
        Object {
          "offset": 14,
          "span": 10,
        }
      }
    >
      <h5>
        Test text
      </h5>
    </Col>
  </Row>
</div>
`;
